applications/integrationtests/mechanics/Feti.cpp

```diff
@@ -367,70 +275,59 @@ std::map<int, VectorXd> ComputeReferenceSolution(int rank)
 
     structure.NodeBuildGlobalDofs(__PRETTY_FUNCTION__);
 
-    structure.GetLogger() << "**********************************************"
-                          << "\n";
-    structure.GetLogger() << "**  real constraints                        **"
-                          << "\n";
-    structure.GetLogger() << "**********************************************"
-                          << "\n\n";
+    structure.GetLogger() << "*********************************** \n"
+                          << "**      real constraints         ** \n"
+                          << "*********************************** \n\n";
 
-    Eigen::VectorXd nodeCoords(2);
-    nodeCoords[0] = 0;
-    nodeCoords[1] = 0;
+    Vector2d coordinate;
+    coordinate[0] = 0;
+    coordinate[1] = 0;
 
-    auto& groupNodesLeftBoundary = structure.GroupGetNodeRadiusRange(nodeCoords, 0, 1.e-6);
+    auto& groupNodesLeftBoundary = structure.GroupGetNodeRadiusRange(coordinate, 0, 1.e-6);
```

Comment:
Why is "LeftBoundary" only one node? In a 2D example, I would expect a line of nodes.